lib/main.py

```diff
@@ -22,8 +22,33 @@ def send_metrics_to_datadog(metrics, tags={}):
     for metric_name, value in metrics.items():
         dd_client.submit_metric(f'lighthouse.{metric_name}', value, tags)
 
-def main():
+def capture_lighthouse_metrics(page_type, url, audits, lighthouse_options=[]):
     lighthouse = Lighthouse()
+    form_factor = 'mobile'
+
+    if '--preset=desktop' in lighthouse_options:
+        form_factor = 'desktop'
+
+    print(f'Running lighthouse for {url} with {form_factor} options\n')
```

Comment:
It won't look as nice, but can we just splat out the entire `lighthouse_options` here?

Reply:
At the moment we only have the `form_factor` option so if we end up adding more lighthouse options then I think splatting them out would be nice to see.